packages/color/test/Color.tests.ts

```diff
@@ -0,0 +1,155 @@
+import { Color } from '@pixi/color';
+
+import type { ColorSource } from '@pixi/color';
+
+describe('Color', () =>
+{
+    it.concurrent('should throw error for invalid color values', async () =>
+    {
+        const invalidColorValues = [
+            -1,
+            '',
+            null,
+            false,
+            true,
+            {},
+            { foo: 'bar' },
+            '0xfffff',
+            '#ff',
+            'ff'
```

Comment:
Maybe add this to the list of invalid colors:

`[1, 1]` & `new Uint8Array([255, 255])`